source/index.html.md.erb

```diff
@@ -1,16 +1,26 @@
 ---
 title: GOV.UK One Login Technical Documentation
 weight: 1
-last_reviewed_on: 2022-09-06
+last_reviewed_on: 2023-01-24
 review_in: 6 months
 ---
 
 # GOV.UK One Login
 
-GOV.UK One Login is the new way for government services to sign in their users.
+GOV.UK One Login is the way for government services to:
 
-GOV.UK One Login will allow users to access any government service using the same username and password.
+* sign in their users
+* prove their users' identity
+
+This technical documentation will help you understand how GOV.UK One Login works. It gives you information on how you:
+
+* plan the functionality your service needs
+* register your service with GOV.UK One Login
+* integrate with GOV.UK One Login to authenticate users and prove their identity
+* go live with your integration
 
 You can [read further documentation about how GOV.UK One Login works](/how-gov-uk-one-login-works/).
 
+Contact us if you have any questions or feedback on our [#govuk-one-login](https://ukgovernmentdigital.slack.com/archives/C02AQUJ6WTC) Slack channel.
```

Comment:
```suggestion
Contact us if you have any questions on our [#govuk-one-login Slack channel](https://ukgovernmentdigital.slack.com/archives/C02AQUJ6WTC).
```
It'd be useful to extend this link so it has more context in isolation 👍 

We don't have a good process in place yet to make sure feedback is picked up on Slack, so I'd like to keep it to questions for now until we have that figured out!